source/includes/_invoices.md

```diff
@@ -170,4 +202,48 @@ Argument | Description
 ---------- | -------
 size | number of items to retrieve
 page | which page to retrieve. _default page size is 10_
-order_by | field used for sorting results
+order_by | field used for sorting results. If missing default is "-due_date". Could be other parameters in the Invoice model, like "-created" or "status=DONE".
+
+## Update an invoice
+> Definition
+
+```
+PUT https://api.shareactor.io/invoices/<invoice_id>
+```
+
+> Example request:
+
+``` http
+PUT /invoices/<invoice_id> HTTP/1.1
+Content-Type: application/json
+Authorization: Bearer <shareactor-api-key>
+X-Share-Session: <session-id>
+Host: api.shareactor.io
+```
+
+Updates an Invoice with a given ID. For example:
+
+Argument | Description
+---------- | -------
+**invoice_id** | ID of the queried Invoice
+account_number | Account number to issuer, to whom the payment of invoice is done to
```

Comment:
account_number & due_date are fields which you can update but they are not arguments for update.
I think ^^